ceph/cephfs/cephfs-provisioner.go

```diff
@@ -345,12 +351,13 @@ func (p *cephFSProvisioner) parsePVSecret(namespace, secretName string) (string,
 }
 
 var (
-	master          = flag.String("master", "", "Master URL")
-	kubeconfig      = flag.String("kubeconfig", "", "Absolute path to the kubeconfig")
-	id              = flag.String("id", "", "Unique provisioner identity")
-	secretNamespace = flag.String("secret-namespace", "", "Namespace secrets will be created in (default: '', created in each PVC's namespace)")
-	enableQuota     = flag.Bool("enable-quota", false, "Enable PVC quota")
-	metricsPort     = flag.Int("metrics-port", 0, "The port of the metrics server (set to non-zero to enable)")
+	master                    = flag.String("master", "", "Master URL")
+	kubeconfig                = flag.String("kubeconfig", "", "Absolute path to the kubeconfig")
+	id                        = flag.String("id", "", "Unique provisioner identity")
+	secretNamespace           = flag.String("secret-namespace", "", "Namespace secrets will be created in (default: '', created in each PVC's namespace)")
+	enableQuota               = flag.Bool("enable-quota", false, "Enable PVC quota")
+	metricsPort               = flag.Int("metrics-port", 0, "The port of the metrics server (set to non-zero to enable)")
+	disableNamespaceIsolation = flag.Bool("disable-namespace-isolation", false, "Disable namespace isolation")
```

Comment:
nit: `disable cephfs namespace isolation` , since namespace itself has too many meanings

Reply:
Fixed. I used `ceph` instead of `cephfs` because other names in this program use `ceph` prefix too.